test/ActivityRepository-test.js

```diff
@@ -1,4 +1,164 @@
-// import { expect } from 'chai'
-// import Activity from '../src/Activity';
-// import ActivityRepository from '../src/ActivtyRepository';
-// import User from '../src/User';
+import { expect } from 'chai'
+import ActivityRepository from '../src/ActivityRepository';
+import User from '../src/User';
+
+
+describe.only('ActivityRepository', function() {
+  let activity1;
+  let activity2;
+  let activity3;
+  let activity4;
+  let activity5;
+  let activity6;
+  let activity7;
+  let mockActivityRepo;
+  let mockRawData; 
+  let todaysDate; 
+  let activityData;
+  let user; 
+
+   beforeEach(() => {
+    activity1 = {
+      userID: 19,
+      date: "2019/06/15",
+      numSteps: 5683,
+      minutesActive: 210,
+      flightsOfStairs: 41
+    },
+    activity2 = {
+      userID: 19,
+      date: "2019/06/16",
+      numSteps: 5439,
+      minutesActive: 236,
+      flightsOfStairs: 22
+    },
+    activity3 = {
+      userID: 19,
+      date: "2019/06/17",
+      numSteps: 5799,
+      minutesActive: 68,
+      flightsOfStairs: 24
+    },
+    activity4 = {
+      userID: 19,
+      date: "2019/06/18",
+      numSteps: 5172,
+      minutesActive: 82,
+      flightsOfStairs: 29
+    },
+    activity5 = {
+      userID: 19,
+      date: "2019/06/19",
+      numSteps: 7614,
+      minutesActive: 290,
+      flightsOfStairs: 18
+    },
+    activity6 = {
+      userID: 19,
+      date: "2019/06/20",
+      numSteps: 3050,
+      minutesActive: 112,
+      flightsOfStairs: 42
+    },
+    activity7 = {
+      userID: 19,
+      date: "2019/06/21",
+      numSteps: 13015,
+      minutesActive: 132,
+      flightsOfStairs: 24
+    },
+    mockRawData = { userData: [ {id: 19,
+      name: "Wilburn Collins",
+      address: "543 Keeling Road, Schummside ID 47123-6269",
+      email: "Blake27@gmail.com",
+      strideLength: 3.3,
+      dailyStepGoal: 8000,
+      friends: [ 30, 43, 22, 39 ]
+    }
+  ],
+    activityData: [activity1, activity2, activity3, activity4, activity5, activity6, activity7]},
+    todaysDate = '06/15/2019', 
+    mockActivityRepo = new ActivityRepository(todaysDate),
+    user = new User ({
+
+    id: 19,
+    name: "Wilburn Collins",
+    address: "543 Keeling Road, Schummside ID 47123-6269",
+    email: "Blake27@gmail.com",
+    strideLength: 3.3,
+    dailyStepGoal: 8000,
+    friends: [30, 43, 22, 39]
+  }, 
+    hydrationInfo = {
+      record: [
+        {
+          "userID": 19,
+          "date": "2019/06/15",
+          "numOunces": 62
+        },
+        {
+          "userID": 19,
+          "date": "2019/06/16",
+          "numOunces": 73
+        }
+      ], 
+      //averageOuncesAllTime: NaN;
+    },
+    sleepInfo = {
```

Comment:
I think if we change these assignment operators to colons the test will pass appropriately. We are trying to create a very large example of a user object, and it has a lot of data, but i've never seen a user object with multiple assignment operators.

Reply:
yeah, that seems foreign to me but, let's see what happens!